src/js/components/Page.js

```diff
@@ -72,10 +71,6 @@ class Page extends mixin(InternalStorageMixin, StoreMixin) {
     this.forceUpdate();
   }
 
-  onSidebarStoreWidthChange() {
```

Comment:
This is actually being used :)
When you subscribe to a store listener using the `StoreMixin` it comes with naming conventions :) Do you remember deleting more like this?